server.js

```diff
@@ -22,11 +14,87 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Homepage
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Netflix titles')
+})
+
+// One title
+app.get('/titles/:id', (req, res) => {
+  const id = req.params.id
+  const title = jsonData.find(item => item.show_id === +id)
+  if (title === undefined) {
+    console.log('empty')
+    res.status(400).send(`There's no matching title`)
+  } else {
+    res.send(title)
+  }
+})
+
+// All titles
+app.get('/titles', paginatedResult(jsonData), (req, res) => {
+  res.send(res.paginatedResult)
+})
+
+// Filtering movies with params
+app.get('/movies', (req, res) => {
+  const movies = jsonData.filter(item => item.type === "Movie")
+  res.send(movies)
+})
+
+app.get('/movies/filter', (req, res) => {
+  const year = req.query.year
+  const movies = jsonData.filter(item => item.release_year === +year)
+  res.send(movies)
+})
+
+// Filtering tv shows with params
+app.get('/tv-shows', (req, res) => {
+  const shows = jsonData.filter(item => item.type === "TV Show")
+  res.send(shows)
 })
 
+app.get('/tv-shows/filter', (req, res) => {
+  const year = req.query.year
+  const shows = jsonData.filter(item => item.release_year === +year)
+  res.send(shows)
+})
```

Comment:
It would be more RESTful if these queries were under the tv-shows endpoint. Endpoints should be named after what they are returning rather than how they are filtered.
`/tv-shows?year=2010`